bd/migrations/01_create_messeges_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE messages (id SERIAL PRIMARY KEY, content TEXT, timestamp TIME );
```

Comment:
You can have the database automatically apply timestamps rather than passing them in with you program objects - when creating a table you can use:

```
CREATE TABLE peeps (
  id SERIAL PRIMARY KEY,
  date_added timestamp DEFAULT CURRENT_TIMESTAMP,
  content varchar(240)
);
```
CURRENT_TIMESTAMP is a PostgreSQL constant that holds the current time.